src/app/dim-ui/destiny-symbols/ColorDestinySymbols.tsx

```diff
@@ -1,4 +1,5 @@
 import { FontGlyphs } from 'data/font/d2-font-glyphs';
+import { DimCustomSymbols } from 'data/font/dim-custom-symbols';
 import styles from './ColorDestinySymbols.m.scss';
 
 const iconPlaceholder = /([\uE000-\uF8FF])/g;
```

Comment:
This regex fails to split the prismatic icon, so the only way this can get colored is if the prismatic icon is the entire string. If you give your Loadout just the prismatic icon as a name, it'll work. I'm also noticing that `#󰅖prismatic` fails to create a hashtag pill, so there's at least one other regex that needs updating.

Reply:
hmm the tag > pill worked for me in that situation, again not colored though
![image](https://github.com/DestinyItemManager/DIM/assets/29002828/d7308538-3ae1-41f7-a599-086013e4f404)

Reply:
Ok I re-checked and I guess I missed it the first time.

Reply:
so basically what you're saying is that is no way to color any of the custom symbols correctly? At least not without some refactoring of how we ID the icons?

Reply:
Depends on your definition of "refactoring". You can try changing the regex to `/([\uE000-\uF8FF\u{F0000}-\u{F1000}])/gu` or something

Reply:
that certainly seems to work 
![image](https://github.com/DestinyItemManager/DIM/assets/29002828/63c374c5-1424-44b7-9969-beee916006e6)